Rakefile

```diff
@@ -498,6 +499,17 @@ file 'build/zlib/libz.a' do
   SH
 end
 
+file 'build/json-c/libjson-c.a' do
+  build_dir = File.expand_path('build/json-c', __dir__)
+  rm_rf build_dir
+  cp_r 'ext/json-c', build_dir
+  sh <<~SH
+    cd #{build_dir} && \
+    cmake . && \
+    make -j 4
+  SH
+end
+
```

Comment:
Is it possible to build without cmake? I'd rather not add another dependency for building Natalie if we don't have to.

Reply:
I thought we already had that dependency, but the CI showed me otherwise.

The cmake command generates a Makefile, so I'm just going to strip that one and include it as an additional file. If that doesn't work, I might just replace this whole implementation with something like nlohmann (or whatever the name was), a C++ header only JSON library.

Reply:
cool cool